FAQ.md

```diff
@@ -51,10 +51,26 @@ Please see the [TOC Principles](https://github.com/cncf/toc/blob/main/PRINCIPLES
+## What is the definition of an adopter?
+Adopters of a CNCF project are any organization that successfully leverages that project in the manner it was intended or repackages it as a core component of a service offering.
+
+The TOC’s intent of identifying adopters is to better understand the operational or production-level use of a given project (which can also be a specification) by its operators or users. We do this to ascertain the level of maturity the project has reached, its interactions with adopters, and its likelihood to continue growing within and supporting the ecosystem.
+
+[Cloud native](https://github.com/cncf/toc/blob/main/DEFINITION.md) project adopters may be any one of the following. Please note that any single company can fall under several categories at once. If that's the case, it should enumerate all that apply and only be listed once. 
+* A CNCF End-User member - Companies and organizations who are End User members of the CNCF (https://www.cncf.io/enduser/ & https://landscape.cncf.io/card-mode?enduser=yes)
+* Another project - an open source project that leverages a CNCF project as part of their solution, integrates with for compatibility and interoperability, or is used in the supply chain of another project
+* End users - companies and organizations that are not CNCF End-User members that use the project and cloud native technologies internally,  or build upon a cloud native open source project but do not sell the cloud native project externally as a service offering (those are Service Providers)
+* A Service Provider - a Service Provider is a organization that repackages an open source project as a core component of a service offering, sells cloud native services externally
+  * APIs
+  * SaaS
+  * A Service Provider’s customers are considered transitive adopters and should be excluded from identification within the ADOPTERS.md file.
+  * Examples of Service Providers (and not end users) include cloud providers (e.g., Alibaba Cloud, AWS, Google Cloud, Microsoft Azure), consulting firms (e.g., Accenture, Booz Allen), infrastructure software vendors (e.g., SUSE, Red Hat), and telecom operators (e.g., AT&T, China Mobile). Refer to the [vendor category on the landscape](https://landscape.cncf.io/category=cncf-members&enduser=no&format=card-mode&grouping=category) for more examples.
+* Consultancy - an entity whose purpose is to assist other organizations in developing a solution leveraging cloud native technology. They may be embedded in the end user team and is responsible for the execution of the service. They may also package cloud native technologies for reuse as part of their offerings. These function as proxies for an end user.
```

Comment:
Here you have consultancy as a category, but above you include Consulting Firms in Service providers.  What's the difference between a consultancy and a consulting firm?

Reply:
Thanks Josh for calling this out, @RichiH did as well and i missed removing this portion of the reference